pycbc/io/live.py

```diff
@@ -388,18 +388,89 @@ def upload(self, fname, gracedb_server=None, testing=True,
                 self.gracedb.write_label(gid, 'INJ')
                 logging.info("Tagging event %s as an injection", gid)
 
-            # add info for tracking code version
-            gracedb_tag_with_version(self.gracedb, gid)
-
-            extra_strings = [] if extra_strings is None else extra_strings
-            for text in extra_strings:
-                self.gracedb.write_log(gid, text, tag_name=['analyst_comments'])
         except Exception as exc:
             logging.error('Something failed during the upload/annotation of '
```

Comment:
Here, since the preceding piece of code only deals with the upload, remove '/annotation' from the error message.